static/schema/extended-schema.schema.json

```diff
@@ -8,11 +8,11 @@
         "concepts": {
           "type": "array",
           "items": {
-            "type": "#/definitions/concept"
+            "type": "http://standardhealthrecord.org/schemas/shr/types.schema.json/definitions/concept"
```

Comment:
I think it's supposed to be `http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/code` (note the `#`).